programmatic-upload-aws-cli.md

```diff
@@ -60,8 +60,12 @@ do
 done
 ```
 
-Once your data is in the bucket, it will automatically be synced with your center’s Synapse project. This process can take anywhere from a few minutes to up to a day depending on the size of your data. Once it is present on Synapse, you can proceed to annotate your metadata.
+Once your data is in the bucket, it will automatically be synced with your center’s Synapse project. This process can take a few minutes depending on the size of your data. Once it is present on Synapse, you can proceed to annotate your metadata.
 
-**Note**: If you would like to make changes to your data, please do so directly from the S3 bucket and not from the Synapse web or programmatic clients. Changes made to the bucket will automatically be updated on the Synapse project.
+### Update Existing Data:
+If you have made changes to a file already in the bucket and would like to upload the new version, please re-upload the modified file to the same S3 key (overwrite the existing file). As with all S3 uploads, ensure that the md5 is included as a metadata flag for larger files. The change will be reflected on Synapse as a new version of the existing Synapse entity.
 
-**Troubleshooting**: If you encounter ‘AccessDenied’ errors, run `aws sts get-caller-identity` to confirm your IAM identity matches that provided to the DCC. If issues persist or for any other questions, open a support ticket [here](https://sagebionetworks.jira.com/servicedesk/customer/portal/1).
+**Note**: Please refrain from moving data in the bucket or adding new versions of data to a separate folder. Doing so will create new Synapse entities for your files, which will require modifying file paths and Synapse IDs in your metadata manifest.
+
+### Troubleshooting: 
+If you encounter ‘AccessDenied’ errors, run `aws sts get-caller-identity` to confirm your IAM identity matches that provided to the DCC. If issues persist or for any other questions, open a support ticket [here](https://sagebionetworks.jira.com/servicedesk/customer/portal/1).
```

Comment:
Can we replace `here` with "with the [HTAN Help Desk](https://sagebionetworks.jira.com/servicedesk/customer/portal/1)"?